src/chess_teacher_stockfish/main.py

```diff
@@ -145,6 +145,7 @@ async def evaluate_position(
                     floor(evaluate_position_request.time_to_analyze)
                 ),
                 wdl_stats=stockfish.get_wdl_stats(),
+                top_three_moves=get_top_three_moves(),
```

Comment:
Just remove the function call and replace it with local logic from `get_top_three_moves` in `evaluate_position`